core/meta/src/TClass.cxx

```diff
@@ -7061,7 +7061,7 @@ TVirtualStreamerInfo *TClass::GetConversionStreamerInfo( const TClass* cl, Int_t
          arr = it->second;
       }
 
-      if( arr && version > -1 && version < arr->GetSize() && arr->At( version ) )
+      if( arr && version >= -1 && version < arr->GetSize() && arr->At( version ) )
```

Comment:
`arr` can handle an index of `-1`?

Reply:
Yes :) ... This category of arrays is created with:
```
new TObjArray(version+10, -1);
```